src/Table/components/commonComponents.tsx

```diff
@@ -67,6 +72,7 @@ export const StyledCell = styled.td<TableStylesProps>`
   ${({ truncateContent }) =>
     truncateContent &&
     css`
+      overflow: hidden;
```

Comment:
Do we want this on truncateContent? I feel like that might be a usecase for tooltips 👀

Reply:
yeah you're right, i'll try find a diff solution - the issue is with this at the moment, for truncateContent is we need the `overflow: hidden` to make use of the `text-overflow: ellipsis` property

if theres nothing super easy, it might make sense to just remove this option & have this logic in the actual column cell as we can customise as much as we want inside there

Reply:
`overflow: clipped` solves it

Reply:
we're so back

Reply:
its over, i got excited and it didn't work - so i think we may aswell keep the prop - but for a tooltip or something like that, we should just handle that logic in the cell